README.md

```diff
@@ -18,21 +18,50 @@ Creating an entry would be ```sprintbot log start mylog```
 
 - It will prompt if all PRs haven't been closed
 
-## Running Locally (not complete yet)
+## Running Locally
 
-clone the repo
+### Start rocket chat
+
+```
+docker run --name db -d mongo:3.0 --smallfiles
+docker run --name rocketchats -p 3001:3000 --env ROOT_URL=http://localhost --link db --link sprintbots -d rocket.chat
+```
+See original steps @ [dockerhub](https://hub.docker.com/_/rocket.chat/)
+
+### set up rocket web hook
+
+Take the following steps to set-up the rocket web hook:
+
+- go to localhost:3001
+- create user
+- click on username on top left
+- Administration -> Integrations -> New Integration -> Outgoing Webhook -> Event Trigger -> Message Sent
+- Set the following
+  - Enabled to True
+  - Select the channel where the sprintbot will be used
+  - Paste to URLs: http://sprintbot:3001/chat/message?source=rocket
```

Comment:
the script runs it as sprintbots and as it is linked into the container as sprintbots the host will be http://sprintbots:3001